vars/Debian.yml

```diff
@@ -0,0 +1,8 @@
+---
+source_list_entries:
```

Comment:
Do we want/need support for DebianSecurity repositories? (`http://archive.debian.org/debian-security`)

Reply:
I'm not sure, but I'm happy to add them if you think we need them.

Reply:
I think we want debian-security.  Please see commit d6c1570.